app/controllers/merchants_controller.rb

```diff
@@ -21,11 +21,6 @@ def create
     end
   end
 
```

Comment:
This came in from the base repo and allowed visitors to delete merchants.  That functionality has been removed